lib/hello_world_web/router.ex

```diff
@@ -3,6 +3,15 @@ defmodule HelloWorldWeb.Router do
 
   pipeline :api do
     plug :accepts, ["json"]
+
+    plug :put_secure_browser_headers, %{
+      "x-xss-protection" => "0",
+      "strict-transport-security" => "max-age=31536000 ; includeSubDomains",
+      "x-frame-options" => "deny",
+      "content-security-policy" => "default-src 'self'; frame-ancestors 'none';",
+      "cache-control" => "no-cache, no-store, max-age=0, must-revalidate",
+      "server" => "undisclosed"
```

Comment:
@manusajith looks good, just one question. Does `server` overrides the value set by the framework?

Reply:
yes, by default it will return: `server: Cowboy`